serializability-checker-compiler-plugin/src/main/scala/org/virtuslab/ash/SerializabilityCheckerCompilerPluginComponent.scala

```diff
@@ -18,52 +20,103 @@ class SerializabilityCheckerCompilerPluginComponent(
     new StdPhase(prev) {
       private val serializabilityTraitType = typeOf[SerializabilityTrait]
 
+      private val genericsWithTypes = Map(
+        ("akka.actor.typed.ActorSystem", Seq(ClassType.Message)),
+        ("akka.actor.typed.ActorRef", Seq(ClassType.Message)),
+        ("akka.actor.typed.Behavior", Seq(ClassType.Message)),
+        ("akka.actor.typed.RecipientRef", Seq(ClassType.Message)),
+        ("akka.persistence.typed.scaladsl.ReplyEffect", Seq(ClassType.PersistentEvent, ClassType.PersistentState)),
+        ("akka.persistence.typed.scaladsl.Effect", Seq(ClassType.PersistentEvent)),
+        ("akka.persistence.typed.scaladsl.EffectBuilder", Seq(ClassType.PersistentEvent)),
+        ("akka.projection.eventsourced.EventEnvelope", Seq(ClassType.PersistentEvent, ClassType.PersistentState)))
+
+      private val genericMethodsWithTypes = Map(
+        ("akka.actor.typed.scaladsl.ActorContext.ask", Seq(ClassType.Message, ClassType.Message)),
+        ("akka.actor.typed.scaladsl.AskPattern.Askable.$qmark", Seq(ClassType.Message)),
+        ("akka.pattern.PipeToSupport.pipe", Seq(ClassType.Message)))
+
+      private val concreteMethodsWithTypes = Map(
+        ("akka.actor.typed.ActorRef.ActorRefOps.$bang", Seq(ClassType.Message)),
+        ("akka.actor.typed.ActorRef.tell", Seq(ClassType.Message)),
+        ("akka.actor.typed.RecipientRef.tell", Seq(ClassType.Message)))
+
+      private val ignoredTypes = List("akka.", "scala.Any", "scala.Nothing")
```

Comment:
Hmmm `akka.` is clearly not a _type_ on its own... maybe this val should be called `ignoredTypePrefixes`?